legacy/src/api/payment-requests/payment-requests.md

```diff
@@ -212,12 +212,15 @@ The Paid By provides a summary of the transactions after the Payment Request was
 
 {% h4 Mandatory Fields %}
 
-|     Field      |        Type        |                               Description                                |
-| -------------- | ------------------ | ------------------------------------------------------------------------ |
-| type           | String             | The asset type used for the payment.                                     |
-| description    | String             | A human readable description of the asset type used.                     |
-| settlementDate | {% dt Timestamp %} | The estimated date that the merchant can expect settlement of funds.     |
-| total          | {% dt Monetary %}  | The total monetary value of the asset type used to pay a Payment Request |
+|     Field      |        Type        |                               Description                                 |
+| -------------- | ------------------ | ------------------------------------------------------------------------- |
+| type           | String             | The asset type used for the payment.                                      |
+| description    | String             | A human readable description of the asset type used.                      |
+| settlementDate | {% dt Timestamp %} | The estimated date that the merchant can expect settlement of funds.      |
+| total          | {% dt Monetary %}  | The total monetary value of the asset type used to pay a Payment Request  |
+| lineItems      | Array {% opt %}    | **Experimental** The [Line Items](#line-item) paid for by the asset type. |
```

Comment:
There is an example response that contains paidBy in this page. Would it be worth adding an example of how line items would look in that example?